airbyte-integrations/bases/bases-destination-jdbc/src/main/java/io/airbyte/integrations/destination/staging/GeneralStagingFunctions.java

```diff
@@ -104,6 +104,7 @@ public static OnCloseFunction onCloseFunction(final JdbcDatabase database,
       // After moving data from staging area to the target table (airybte_raw) clean up the staging
       // area (if user configured)
       log.info("Cleaning up destination started for {} streams", writeConfigs.size());
+      typerDeduper.typeAndDedupe();
```

Comment:
@benmoriceau / @tryangul fyi we're tweaking the T+D interface again 😅 . There's now a `typeAndDedupe()` method, which runs T+D on all streams, along with a new `cleanup` method (under the hood, it shuts down a thread pool, and is mandatory to run at the end of a sync to allow the process to exit)